v2/runner/flags.go

```diff
@@ -137,6 +139,10 @@ func newFlagParser(name string, pType types.PluginType, opt *plugin.Options) *fl
 			"debug-collect-interval", defaultCollectInterval,
 			"Interval between consecutive collect requests")
 
+		flagParser.IntVar(&opt.CollectChunkSize,
```

Comment:
I would update ValidateOptions(opt *plugin.Options) error
Currently you can probably pass 0 / negative value

Reply:
I left such possibility intentionally but you're right this doesn't make sense as that value is passed as a channel buffer size

Reply:
done